### PR TITLE
cherokee, bump version, remove old bep/patch files, import work from …

### PR DESCRIPTION
**Pull request by Begasus**
…arfonzo

### CODE REVIEW

File: www-servers/cherokee/cherokee-1.2.104.recipe [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	cherokee$secondaryArchSuffix = $portVersion
+	cmd:ctk_run$secondaryArchSuffix
+	cmd:cget$secondaryArchSuffix
+	cmd:cherokee$secondaryArchSuffix
+	cmd:cherokee_admin$secondaryArchSuffix
+	cmd:cherokee_admin_launcher$secondaryArchSuffix
+	cmd:cherokee_config$secondaryArchSuffix
```
Comment (korli) [inline www-servers/cherokee/cherokee-1.2.104.recipe]: cherokee-config should go in the devel package.

File: www-servers/cherokee/cherokee-1.2.104.recipe [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+	settings/cherokee/cherokee.conf.perf_sample keep-old
+	settings/cherokee/cherokee.conf keep-old
+	var/www/images/cherokee-logo.png keep-old
+	var/www/images/default-bg.png keep-old
+	var/www/images/favicon.ico keep-old
+	var/www/images/powered_by_cherokee.png keep-old
+	var/www/index.html keep-old
+	"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
```
Comment (korli) [inline www-servers/cherokee/cherokee-1.2.104.recipe]: I'd drop x86_gcc2 if it works for x86 as secondary arch. Or remove the secondary arch.
The libraries are only useful for the arch running the server.

File: www-servers/cherokee/cherokee-1.2.104.recipe [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+		libcherokee-client \
+		libcherokee-server
+
+	fixPkgconfig
+
+	packageEntries devel \
+		$developDir
+
+	echo "Copying files from ${prefix}/share/cherokee/ ..."
+	cp -R ${prefix}/share/cherokee/* ${dataDir}/cherokee/
```
Comment (korli) [inline www-servers/cherokee/cherokee-1.2.104.recipe]: Braces are unneeded.

**Comment by Begasus** [comment]
based on @arfonzo's work (not tested on x86_64 yet)

**Comment by Begasus** [comment]
cherokee

**Comment by Begasus** [comment]
cherokee2
cherokee3

**Comment by korli** [comment]
@Begasus merge?

**Comment by Begasus** [comment]
@korli looks ok to me, some issues probably will come up, but as it looks it's working

**Pull request merged**